rapyuta_io_sdk_v2/config.py

```diff
@@ -86,7 +86,10 @@ def get_headers(
         Returns:
             dict: Headers for the configuration.
         """
-        headers = dict(Authorization=f"Bearer {self.auth_token}")
+        auth_value = self.auth_token
```

Comment:
suggestion: auth_value = self.auth_token.strip() if self.auth_token else None

if auth_value and not auth_value.lower().startswith("bearer "):
    auth_value = f"Bearer {auth_value}"

headers = {"Authorization": auth_value} if auth_value else {}